spec/static/index.html

```diff
@@ -17,6 +17,12 @@
   const electron = require('electron')
   const { remote, ipcRenderer } = electron
 
+  // Set up chai-as-promised here first to avoid conflicts
+  // It must be loaded first or really strange things happen inside
+  // chai that cause test failures
+  // DO NOT MOVE, REMOVE OR EDIT THIS LINE
```

Comment:
it just needs to be set up _before_ dirty-chai right?

can we just `.use()` them both in this file and remove all the `.use()`s from other places? that'd let us have a less scary comment here

Reply:
@nornagon Yes, mocha actually has a built in way to do this with a "setup" file.  I'll add this in after this PR in a follow up 👍